.github/workflows/ci.yml

```diff
@@ -26,10 +26,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Install Python 3.10
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
```

Comment:
No need to install 3.10. Just change `linters-py310` to `linters-py312`, unless that causes a bunch of unexpected linting errors we'll have to work out separately.

Reply:
Not sure about the builder itself but on other project we try to keep it to the minimum supported python as some linters or formatters might request or make incompatible if run with newer pythons. Even so, nowadays is much better, will do.